tools/api-surface/Tests/APISurfaceCoreTests/IntegrationTests.swift

```diff
@@ -6,71 +6,155 @@
 
 import XCTest
 import APISurfaceCore
+import ArgumentParser
 
 class IntegrationTests: XCTestCase {
     func testCreatingSurfaceForFixtureLibraries() throws {
-        var outputs: [String] = []
-        APISurfaceCore.printFunction = { outputs.append($0) }
-
         // Given
-        let command = try SPMLibrarySurfaceCommand.parse([
+        let temporaryFile = "/tmp/api-surface-test-output"
```

Comment:
suggestion/ Using a dynamically created temp directory could bring some benefits:
- Avoid conflicts between test runs and ensure isolation.
- Automatically clean up after tests to prevent leftover files from affecting others.
- More robust (in other environments) than using hardcoded paths.

Example:
```swift
let tempDirectory = FileManager.default.temporaryDirectory.appendingPathComponent("com.datadoghq.api-surface-" + UUID().uuidString)
try FileManager.default.createDirectory(at: tempDirectory, withIntermediateDirectories: true, attributes: nil)
        
let temporaryFile = tempDirectory.appendingPathComponent("api-surface-test-output").path
```